pkg/config/amqonline.go

```diff
@@ -112,6 +112,10 @@ func (a *AMQOnline) GetBackupsSecretName() string {
 	return "backups-s3-credentials"
 }
 
+func (c *AMQOnline) GetPostgresBackupSecretName() string {
+	return "enmasse-postgres-secret"
```

Comment:
Just wondering is the meant to backup the `standard-auth-service` postgres?
Cause this secret doesn't exist or created by CRO? :thinking: 

https://github.com/integr8ly/integreatly-operator/blob/c3ce78c1abf241cb3de792c1b3b7acf7fefe6022/pkg/products/amqonline/reconciler.go#L275
https://github.com/integr8ly/integreatly-operator/blob/c3ce78c1abf241cb3de792c1b3b7acf7fefe6022/pkg/products/amqonline/reconciler.go#L304

So running the backup job would fail :confused: 
```
oc create job enmasse-postgres-backup-$(date +"%Y-%m-%d-%H%M%S") --from=cronjob/enmasse-postgres-backup -n redhat-rhmi-amq-online
```

Reply:
:eyes:

Reply:
/hold

Reply:
good catch , the secret needs to be created like
https://github.com/integr8ly/integreatly-operator/blob/master/pkg/products/codeready/reconciler.go#L219L241

Reply:
Hmm yeah, I'm not too sure also, maybe just changing `GetPostgresBackupSecretName()` to return `standard-authservice-postgresql` would do :confused: 

@grdryn Might be able to better comment on this since he added the change for the in cluster postgres of amq online

Reply:
`standard-authservice-postgresql` doesn't connect to psql
```sh
psql: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
==> Component data dump completed
ls: cannot access /tmp/intly/archives/*: No such file or directory```

Reply:
If you're going to change this to return `standard-authservice-postgresql`, then change https://github.com/integr8ly/integreatly-operator/blob/c3ce78c1abf241cb3de792c1b3b7acf7fefe6022/pkg/products/amqonline/reconciler.go#L275 to get the value from this function rather than having two different places where it's defined. 

Also maybe change the function name since it wouldn't be specific to backups then.

Reply:
@austincunningham is the secret formatted in the expected way for the backup script?

Reply:
@grdryn I suspect you are correct the format is wrong , not quite sure where to find the correct format, but the previous example in 
codeready-postgres-secret  which has
```yaml
data:
  POSTGRES_DATABASE: 
  POSTGRES_HOST: 
  POSTGRES_PASSWORD: 
  POSTGRES_PORT: 
  POSTGRES_USERNAME: 
  POSTGRES_VERSION: 
``` 


where as standard-authservice-postgresql has 
```yaml
data:
  database-password: 
  database-user: 
```

Reply:
Yeah, that looks correct: https://github.com/integr8ly/backup-container-image/blob/0d0fb63707f3485fa22c1fc2e109bb185da01e2c/image/tools/lib/component/postgres.sh#L3..L25

I think you'll need to either create a new secret, or maybe check the secret that the cloud-resource-operator creates in the redhat-rhmi-operator repo...maybe that's in the right format?